src/verilog.y

```diff
@@ -5319,10 +5319,10 @@ concurrent_assertion_item<nodep>:       // IEEE: concurrent_assertion_item
 concurrent_assertion_statement<nodep>:  // ==IEEE: concurrent_assertion_statement
         //                      // IEEE: assert_property_statement
         //UNSUP remove below:
-                yASSERT yPROPERTY '(' property_spec ')' elseStmtBlock   { $$ = new AstAssert($1, $4, nullptr, $6, false); }
+                yASSERT yPROPERTY '(' property_spec ')' elseStmtBlock   { $$ = new AstAssert($1, new AstSampled($1, $4), nullptr, $6, false); }
```

Comment:
```suggestion
                yASSERT yPROPERTY '(' property_spec ')' elseStmtBlock
                        { $$ = new AstAssert{$1, new AstSampled{$1, $4}, nullptr, $6, false}; }
```

Reply:
I've reformatted this rule and the other one (72031e2797e28d448f67c958c89ea754b914de00).